src/components/ButtonLink.js

```diff
@@ -42,19 +43,23 @@ const Secondary = styled(StyledButton)`
   }
 `
 
-const ButtonLink = ({ to, isSecondary, children, className }) => {
+const ButtonLink = ({ to, isSecondary, children, className, ...props }) => {
   if (isSecondary) {
     return (
-      <Secondary to={to} hideArrow={true} className={className}>
+      <Secondary to={to} hideArrow={true} className={className} {...props}>
         {children}
       </Secondary>
     )
   }
   return (
-    <Primary to={to} hideArrow={true} className={className}>
+    <Primary to={to} hideArrow={true} className={className} {...props}>
       {children}
     </Primary>
   )
 }
 
+ButtonLink.defaultProps = {
+  mt: `0.5rem`,
```

Comment:
@ryancreatescopy curious for your thoughts on this. I added this to keep parity with the current designs (default margin-top of 0.5rem on all buttons) but I don't think this is actually needed/desired. Seems kind of strange to me to apply a margin to all buttons.

Reply:
The desired margin will probably differ dependent on context. So it'd probably be best to remove a default margin IMO as you'll need to add custom margin regardless.